src/sa_web/static/js/views/place-counter-view.js

```diff
@@ -7,12 +7,14 @@ var Shareabouts = Shareabouts || {};
     initialize: function() {
```

Comment:
We should probably re-implement this to take an array of dataset id's (or collection of datasets), and only count the places associated with those datasets.